includes/Modules/Idea_Hub.php

```diff
@@ -434,7 +434,7 @@ function( $post_id ) {
 					)
 				);
 			case 'POST:create-idea-draft-post':
-				$idea = $data['idea'];
+				$idea = $this->get_post_idea( $response );
```

Comment:
@ivankruchkoff let's better combine `GET:draft-post-ideas` and `POST:create-idea-draft-post` cases since they do the same thing. Just do it like this:

```diff
 switch ( "{$data->method}:{$data->datapoint}" ) {
+    case 'POST:create-idea-draft-post':
     case 'GET:draft-post-ideas':
         return array_filter(
             array_map(
                 function( $post_id ) {
                     return array_merge(
                         array(
                             'postID'      => $post_id,
                             'postEditURL' => get_edit_post_link( $post_id ),
                         ),
                         $this->get_post_idea( $post_id )
                     );
                 },
                 is_array( $response ) ? $response : array( $response )
             )
         );
     case 'GET:published-post-ideas':
         ...
-    case 'POST:create-idea-draft-post':
-        $idea = $this->get_post_idea( $response );
-        return array(
-            ...
-        );
```